documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces.adoc

```diff
@@ -1215,8 +1215,8 @@ h|Name h|Description h|Mand. h|Type h|Card.
 |key |The key of the AssetId The name of the specific asset identifier or the predefined name “_globalAssetId_” that would refer to the _AssetInformation/globalAssetId_ |yes |string |1
 |keyIdentifier |The key identifier object |yes |string |1
 |serializationModifier |Defines the format of the response |yes |xref:specification/interfaces-operation-parameters.adoc#SerializationModifier[SerializationModifier]  |1
-|limit |The maximum size of the result set |yes |nonNegativeInteger |1
-|cursor |The position from which to resume a result listing |yes |string |1
+|limit |The maximum size of the result set |no |nonNegativeInteger |1
+|cursor |The position from which to resume a result listing |no |string |1
```

Comment:
Here we changed the mandatory field as nowhere else the limit and cursor were mandatory